### PR TITLE
Fixes reduntant onClick handling

### LINKED ISSUE #42: Redundant 'onClick' handling
## Issue Description
'onClick' handler is used two times for one element inside Navbar (Navbar.jsx) when logging out. It may lead to conflicts in future.

## Steps to Reproduce (if applicable)

1. Clone the repo and check Navbar.jsx

## Current Behavior
image
image

### PR DESCRIPTION
**Pull request by itshypon**
Fixed #42 
Added both methods in one onClick handler, things seem to work fine.
Due to two onClick handlers and looking at precedence, only handler below is executed. Now, both of them works as both methods are added in one handler.
Still, give a check if something else is broken although everything seems fine.

### CODE REVIEW

File: components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -268,11 +268,14 @@ const Navbar = () => {
...
-              onClick={(r) => setNav('logout')}
+              onClick={(r) =>{
+                 setNav('logout');
+                 logout();
+              }}
               style={{
                 color: `${nav === 'logout' ? 'rgb(218, 84, 84)' : 'gray'}`,
               }}
-              onClick={() => logout()}
+              // onClick={() => logout()}
```
Comment (varun-singhh) [inline components/Navbar/Navbar.jsx]: instead of commenting remove the dead code

File: components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -337,11 +340,14 @@ const Navbar = () => {
...
-              onClick={(r) => setNav('logout')}
+              onClick={(r) =>{
+               setNav('logout');
+               onOpenModal();
+              }}
               style={{
                 color: `${nav === 'logout' ? 'rgb(218, 84, 84)' : 'gray'}`,
               }}
-              onClick={onOpenModal}
+              // onClick={onOpenModal}
```
Comment (varun-singhh) [inline components/Navbar/Navbar.jsx]: remove dead code

File: components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -337,11 +340,14 @@ const Navbar = () => {
...
             >
               {' '}
               &nbsp;
```
Comment (varun-singhh) [inline components/Navbar/Navbar.jsx:350]: remove `{' '}`

**Review by varun-singhh — CHANGES_REQUESTED** [review]

**Comment by itshypon** [comment]
Done @varun-singhh, mistakenly wrote same message for commit though.

**Review by varun-singhh — APPROVED** [review]

**Pull request merged**